docs/src/configuration/app/build.md

```diff
@@ -80,34 +111,72 @@ hooks:
-* `/tmp` - The temp directory is also useful for writing files that are not needed in the final application, but it will be wiped between each build.
-
-There are no constraints on what can be downloaded during your build hook except for the amount of disk available at that time. Independent of the mounted disk size you have allocated for deployment, build environments (the application plus the cache directory) and therefore application images are limited to 4 GB during the build phase. If you exceed this limit you will receive a `No space left on device` error. It is possible to increase this limit in certain situations, but it will be necessary to open a support ticket in order to do so. Consult the [Troubleshooting](/development/troubleshoot.md#no-space-left-on-device) guide for more information on this topic.
+The `build` hook is run after the build flavor (if any).
+At this point no services (such as a database) are available nor any persistent file mounts,
+as the application hasn't yet been deployed.
+Environment variables that exist only at runtime such as `PLATFORM_BRANCH` and `PLATFORM_DOCUMENT_ROOT` anre't available during this phase.
+See the full list of provided [build time and runtime variables](/development/variables.md#platformsh-provided-variables).
+There are three writeable directories at this time:
+
+* `$PLATFORM_APP_DIR`:
+  This is where your code is checked out and is the working directory when the build hook starts.
+  The contents of this directory after the build hook is the application that gets deployed.
+  (This directory is always `/app`, but it's better to use the variable or rely on the working directory than to hard code that.)
+  Most of the time, this is the only directory you use.
+* `$PLATFORM_CACHE_DIR`:
+  This directory persists between builds, but is NOT deployed as part of your application.
+  It's a good place for temporary build artifacts, such as downloaded `.tar.gz` or `.zip` files, that can be reused between builds.
+  Note that it's shared by all builds on all branches,
+  so if using the cache directory make sure your build code accounts for that.
+* `/tmp`:
+  The temp directory is also useful for writing files that aren't needed in the final application,
+  but it's wiped between each build.
+
+There are no constraints on what can be downloaded during your build hook except for the amount of disk available at that time.
+Independent of the mounted disk size you have allocated for deployment,
+build environments (the application plus the cache directory) and therefore application images are limited to 4 GB during the build phase.
+If you exceed this limit, you receive a `No space left on device` error.
+It's possible to increase this limit in certain situations,
+but it's necessary to open a support ticket to do so.
```

Comment:
Link to the support dashboard/ticket? In the page linked the following line, couldn't see a link either: "If for some reason your application requires more than 4 GB during build, you can open a support ticket to have this limit increased. The most disk space available during build still caps off at 8 GB in these cases."